src/services/accommodationService.js

```diff
@@ -0,0 +1,75 @@
+import models from '../database/models';
+
+const { Accommodation } = models;
+/** Class representing accommodation services */
+export default class AccommodationServices {
+  /**
+   * @description get accommodation by name if exists
+   * @param {string} data parameter to be checked against
+   * @return {object} Oject of user if found
+   */
+  static async findAccommodation(data) {
+    try {
+      const accommodation = await Accommodation.findOne({ where: data });
+      return accommodation || null;
```

Comment:
Kindly take time and look into another service to see how others are being implemented.